eng/Directory.Packages.props

```diff
@@ -293,13 +293,14 @@
     <PackageVersion Include="runtime.linux-x64.Microsoft.NETCore.ILAsm" Version="$(ILAsmPackageVersion)" />
     <PackageVersion Include="runtime.osx-x64.Microsoft.NETCore.ILAsm" Version="$(ILAsmPackageVersion)" />
     <PackageVersion Include="Basic.CompilerLog.Util" Version="0.6.1" />
-    <PackageVersion Include="Basic.Reference.Assemblies.NetStandard20" Version="1.6.0" />
-    <PackageVersion Include="Basic.Reference.Assemblies.Net50" Version="1.6.0" />
-    <PackageVersion Include="Basic.Reference.Assemblies.Net60" Version="1.6.0" />
-    <PackageVersion Include="Basic.Reference.Assemblies.Net70" Version="1.6.0" />
-    <PackageVersion Include="Basic.Reference.Assemblies.Net80" Version="1.4.5" />
-    <PackageVersion Include="Basic.Reference.Assemblies.Net461" Version="1.6.0" />
-    <PackageVersion Include="Basic.Reference.Assemblies.NetStandard13" Version="1.6.0" />
+    <PackageVersion Include="Basic.Reference.Assemblies.NetStandard20" Version="1.7.2" />
```

Comment:
>1.7.2 [](http://example.com/codeflow?start=80&length=5)

Would it make sense to abstract this version number so that we can upgrade from just one location?

Reply:
Yeah that would make sense now that I've unified them all.